org.ekstep.questionunit.sequence-1.0/renderer/js/seq-template.js

```diff
@@ -77,9 +77,23 @@ SEQController.touchConvertInit = function() {
 SEQController.onDomReady = function () {
   $(document).ready(function () {
     SEQController.touchConvertInit();
-    $(".option-block-container").sortable();
+    $(".option-block-container").sortable({
+      start: function(evt,ui){
+
+      },
+      stop: function(evt,ui){
+        SEQController.logTelemetryInteract(evt);
+      }
+    });
     $(".option-block-container").disableSelection();
-  })
-}
+  });
+};
+
+SEQController.logTelemetryInteract = function (event) {
+  QSTelemetryLogger.logEvent(QSTelemetryLogger.EVENT_TYPES.TOUCH, {
```

Comment:
Are you generating event type `TOUCH`?
If yes remove and change it to `DROP` event type should be `DROP`